src/Microsoft.AspNetCore.SignalR.Redis/RedisHubLifetimeManager.cs

```diff
@@ -288,6 +312,23 @@ public void Dispose()
             }
         }
 
+        private string GetInvocationId()
+        {
+            var invocationId = Interlocked.Increment(ref _nextInvocationId) - 1;
```

Comment:
I think it makes perfect sense for it to be in the lifetime manager. The lifetime manager handles all invocations from server to client, right?